modules/dailymotionBidAdapter.js

```diff
@@ -0,0 +1,56 @@
+import { config } from '../src/config.js';
+import { registerBidder } from '../src/adapters/bidderFactory.js';
+import { VIDEO } from '../src/mediaTypes.js';
+
+export const spec = {
+  code: 'dailymotion',
+  gvlid: 573,
+  supportedMediaTypes: [VIDEO],
+
+  /**
+   * Determines whether or not the given bid request is valid.
+   * The only mandatory parameters for a bid to be valid are the api_key and position configuration entries.
+   * Other parameters are optional.
+   *
+   * @return boolean True if this is a valid bid, and false otherwise.
+   */
+  isBidRequestValid: () => {
+    const dmConfig = config.getConfig('dailymotion');
+    return !!(dmConfig?.api_key && dmConfig?.position);
+  },
+
+  /**
+   * Make a server request from the list of valid BidRequests (that already passed the isBidRequestValid call)
+   *
+   * @param {BidRequest[]} validBidRequests A non-empty list of valid bid requests that should be sent to the Server.
+   * @param {BidderRequest} bidderRequest
+   * @return ServerRequest Info describing the request to the server.
+   */
+  buildRequests: (validBidRequests, bidderRequest) => (validBidRequests || []).map(bid => ({
```

Comment:
We could have something like that `buildRequests: (validBidRequests = [], bidderRequest) => validBidRequests.map(bid => ({` but as you did it works 👍🏻